lib/utilities/get-script-file-name.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function getScriptFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
+  return Array.isArray(filePath.match(scriptFileNameRegEx)) ? filePath.match(scriptFileNameRegEx)[0] : '';
+}
```

Comment:
I think if we use `jsdom` we can directly use APIs like `getAttribute('src')` which might eliminate the need for the regex.

Reply:
see above comment